companies/ratp.json

```
@@ -0,0 +1,37 @@
+{
+    "slug": "ratp",
+    "relevant-countries": [
+        "fr"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Régie Autonome des Transports Parisiens (RATP)",
+    "address": "185 rue de Bercy, LT73\n75012 Paris\nFrance",
+    "email": "protection-donnees@ratp.fr",
+    "web": "https://ratp.fr",
+    "sources": [
+        "https://www.ratp.fr/en/ratp-general-privacy-policy",
+        "https://github.com/datenanfragen/data/pull/1373"
+    ],
+    "required-elements": [
+        {
+            "desc": "Name",
+            "type": "name",
+            "optional": false
+        },
+        {
+            "desc": "Email",
+            "type": "email",
+            "optional": false
+        },
+        {
+            "desc": "ID of Navigo pass (if holder)",
+            "type": "input",
+            "optional": true
+        }
+    ],
+    "suggested-transport-medium": "email",
+    "quality": "verified",
+    "nsfw": false
```

This field was introduced in #1367 and should only be set if `true`. We had a brief bug where the suggest page automatically added it (fixed in datenanfragen/website#716 and #1375). In future PRs, the tests should catch this as an error.

```suggestion
    "quality": "verified"
```